index.html

```diff
@@ -48,7 +81,7 @@ <h4>
         {% for post in site.tags.release limit:3 %}
         <p class="text-faded mb-2">{{ post.date | date_to_string }}: <a href="{{ post.download }}"><strong>WPE WebKit {{ post.version }} {{ post.category | capitalize }}</strong></a></p>
```

Comment:
We still use here `post.category`, is this going to work?

Reply:
Ah, I see that `feed.xml` also uses categories, but it won't break things because it has a loop, so it won't generate any output if there are no categories:

```jinja
        {% for cat in post.categories %}
        <category>{{ cat | xml_escape }}</category>
        {% endfor %}
```

Reply:
> We still use here `post.category`, is this going to work?

It does not break anything, but the strings “Stable” and “Unstable” are not added to the text of the links. Do we still want to have this column here with the downloads, now that we have all the stable and unstable releases in the table above? I think not, because listing just tree downloads here will never fit as many as the table, so I think we can move the “More...” button next to the table and just kill this column here. WDYT?

Reply:
Sounds good.

Reply:
I'm going to go ahead and remove the column. I have noticed that the “Downloads” column has hardcoded the “WPE WebKit” text for all releases, so that would be wrongly used also for libwpe and WPEBackend-fdo :unamused: